### ISSUE TITLE
The game crashed when I right click a leadplate block with a bucket.

### ISSUE THREAD

**Issue by CyberKanjousen** [CannoliCatfish/project-rankine#329]
This is the crash reports:
```
---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 13/11/2022 下午4:32
Description: Unexpected error

java.lang.IllegalArgumentException: Cannot get property BooleanProperty{name=waterlogged, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{rankine:lead_sheetmetal}
	at net.minecraft.world.level.block.state.StateHolder.m_61143_(StateHolder.java:98) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:ferritecore.fastmap.mixin.json:FastMapStateHolderMixin,pl:mixin:A}
	at net.minecraft.world.level.block.SimpleWaterloggedBlock.m_142598_(SimpleWaterloggedBlock.java:37) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:classloading}
	at net.minecraft.world.item.BucketItem.m_7203_(BucketItem.java:70) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:classloading,xf:fml:forge:bucketitem}
	at net.minecraft.world.item.ItemStack.m_41682_(ItemStack.java:252) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,xf:fml:forge:itemstack,re:classloading,xf:fml:forge:itemstack,pl:mixin:APP:tetra.mixins.json:MixinItemStack,pl:mixin:A}
	at net.minecraft.client.multiplayer.MultiPlayerGameMode.m_105235_(MultiPlayerGameMode.java:336) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91277_(Minecraft.java:1595) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:MinecraftClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91279_(Minecraft.java:1852) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:MinecraftClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91398_(Minecraft.java:1669) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:MinecraftClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1020) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:MinecraftClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:664) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:MinecraftClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.1.73.jar%2317!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.world.level.block.state.StateHolder.m_61143_(StateHolder.java:98) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:ferritecore.fastmap.mixin.json:FastMapStateHolderMixin,pl:mixin:A}
	at net.minecraft.world.level.block.SimpleWaterloggedBlock.m_142598_(SimpleWaterloggedBlock.java:37) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:classloading}
	at net.minecraft.world.item.BucketItem.m_7203_(BucketItem.java:70) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:classloading,xf:fml:forge:bucketitem}
	at net.minecraft.world.item.ItemStack.m_41682_(ItemStack.java:252) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,xf:fml:forge:itemstack,re:classloading,xf:fml:forge:itemstack,pl:mixin:APP:tetra.mixins.json:MixinItemStack,pl:mixin:A}
	at net.minecraft.client.multiplayer.MultiPlayerGameMode.m_105235_(MultiPlayerGameMode.java:336) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91277_(Minecraft.java:1595) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:MinecraftClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91279_(Minecraft.java:1852) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:MinecraftClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
-- Affected level --
Details:
	All players: 1 total; [LocalPlayer['dwb_CN'/57237, l='ClientLevel', x=539443.70, y=29.00, z=296031.63]]
	Chunk stats: 1024, 461
	Level dimension: minecraft:overworld
	Level spawn location: World: (27,169,31), Section: (at 11,9,15 in 1,10,1; chunk contains blocks 16,-64,16 to 31,319,31), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 92652823 game time, 93194829 day time
	Server brand: forge
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.ClientLevel.m_6026_(ClientLevel.java:407) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:dynamiclightsreforged.mixins.json:ClientWorldMixin,pl:mixin:APP:rubidium.mixins.json:features.chunk_rendering.MixinClientWorld,pl:mixin:APP:rubidium.mixins.json:features.fast_biome_colors.MixinClientWorld,pl:mixin:APP:flywheel.mixins.json:ClientLevelMixin,pl:mixin:APP:mixins.oculus.vertexformat.json:block_rendering.MixinClientLevel,pl:mixin:APP:create.mixins.json:DestroyProgressMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91354_(Minecraft.java:2261) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:MinecraftClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:686) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:MinecraftClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.18.2-20220404.173914-srg.jar%2386!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.1.73.jar%2317!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?] {}


-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Mod Resources, 彩虹像素RainbowPixel_☆v2.7.zip

-- System Details --
Details:
	Minecraft Version: 1.18.2
	Minecraft Version ID: 1.18.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
	Memory: 1231578688 bytes (1174 MiB) / 2315255808 bytes (2208 MiB) up to 2315255808 bytes (2208 MiB)
	CPUs: 8
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
	Identifier: Intel64 Family 6 Model 142 Stepping 10
	Microarchitecture: Coffee Lake
	Frequency (GHz): 1.80
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: Radeon (TM) RX 550
	Graphics card #0 vendor: Advanced Micro Devices, Inc. (0x1002)
	Graphics card #0 VRAM (MB): 2048.00
	Graphics card #0 deviceId: 0x699f
	Graphics card #0 versionInfo: DriverVersion=30.0.14030.0
	Graphics card #1 name: Intel(R) UHD Graphics 620
	Graphics card #1 vendor: Intel Corporation (0x8086)
	Graphics card #1 VRAM (MB): 1024.00
	Graphics card #1 deviceId: 0x5917
	Graphics card #1 versionInfo: DriverVersion=27.20.100.9749
	Memory slot #0 capacity (MB): 4096.00
	Memory slot #0 clockSpeed (GHz): 2.40
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 4096.00
	Memory slot #1 clockSpeed (GHz): 2.40
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 18052.52
	Virtual memory used (MB): 15966.29
	Swap memory total (MB): 9963.98
	Swap memory used (MB): 1712.27
	JVM Flags: 12 total; -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=16m -XX:-UseAdaptiveSizePolicy -XX:-OmitStackTraceInFastThrow -XX:-DontCompileHugeMethods -Xmn128m -Xmx2204m -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump
	Loaded Shaderpack: (off)
	Launched Version: 1.18.2
	Backend library: LWJGL version 3.2.2 SNAPSHOT
	Backend API: Intel(R) UHD Graphics 620 GL version 3.2.0 - Build 27.20.100.9749, Intel
	Window size: 1366x697
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, mod_resources, file/彩虹像素RainbowPixel_☆v2.7.zip (incompatible)
	Current Language: 简体中文 (中国)
	CPU: 8x Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
	ModLauncher: 9.1.3+9.1.3+main.9b69c82a
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		 mixin PLUGINSERVICE 
		 eventbus PLUGINSERVICE 
		 slf4jfixer PLUGINSERVICE 
		 object_holder_definalize PLUGINSERVICE 
		 runtime_enum_extender PLUGINSERVICE 
		 capability_token_subclass PLUGINSERVICE 
		 accesstransformer PLUGINSERVICE 
		 runtimedistcleaner PLUGINSERVICE 
		 mixin TRANSFORMATIONSERVICE 
		 fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@null
		javafml@null
	Mod List: 
		client-1.18.2-20220404.173914-srg.jar             |Minecraft                     |minecraft                     |1.18.2              |DONE      |Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		tetra-1.18.2-4.9.3_.jar                           |Tetra                         |tetra                         |4.9.3               |DONE      |Manifest: NOSIGNATURE
		dynamiclightsreforged-1.18.2_v1.3.3.jar           |Rubidium Dynamic Lights       |dynamiclightsreforged         |1.18.2_v1.3.3       |DONE      |Manifest: NOSIGNATURE
		voicechat-forge-1.18.2-2.3.2_.jar                 |Simple Voice Chat             |voicechat                     |1.18.2-2.3.2        |DONE      |Manifest: NOSIGNATURE
		createbigcannons-1.18.2-beta-0.3_.jar             |Create Big Cannons            |createbigcannons              |1.18.2-beta-0.3     |DONE      |Manifest: NOSIGNATURE
		create_misc_and_things_1.18.2_0.1a_.jar           |create: things and misc       |create_things_and_misc        |1.0.0               |DONE      |Manifest: NOSIGNATURE
		Jade-1.18.2-forge-5.2.3_.jar                      |Jade                          |jade                          |5.2.3               |DONE      |Manifest: NOSIGNATURE
		CMDCam_FORGE_v2.0.5_mc1.18.2.jar                  |CMDCam                        |cmdcam                        |2.0.5               |DONE      |Manifest: NOSIGNATURE
		CreativeCore_FORGE_v2.6.12_mc1.18.2.jar           |CreativeCore                  |creativecore                  |2.6.12              |DONE      |Manifest: NOSIGNATURE
		crafting-dead-core-1.18.2-1.7.6-alpha_.jar        |Crafting Dead                 |craftingdead                  |1.7.6-alpha         |DONE      |Manifest: NOSIGNATURE
		jei-1.18.2-9.7.1.232_.jar                         |Just Enough Items             |jei                           |9.7.1.232           |DONE      |Manifest: NOSIGNATURE
		spark-1.9.11-forge_.jar                           |spark                         |spark                         |1.9.11              |DONE      |Manifest: NOSIGNATURE
		rubidium-0.5.3a.jar                               |Rubidium                      |rubidium                      |0.5.3a              |DONE      |Manifest: NOSIGNATURE
		flywheel-forge-1.18-0.6.4_.jar                    |Flywheel                      |flywheel                      |1.18-0.6.4.87       |DONE      |Manifest: NOSIGNATURE
		Patchouli-1.18.2-71.1_.jar                        |Patchouli                     |patchouli                     |1.18.2-71.1         |DONE      |Manifest: NOSIGNATURE
		mutil-1.18.2-4.5.0_.jar                           |mutil                         |mutil                         |4.5.0               |DONE      |Manifest: NOSIGNATURE
		create-mc1.18.2_v0.5.0c_.jar                      |Create                        |create                        |0.5.0c              |DONE      |Manifest: NOSIGNATURE
		oculus-mc1.18.2-1.2.5a.jar                        |Oculus                        |oculus                        |1.2.5a              |DONE      |Manifest: NOSIGNATURE
		TerraForged-1.18.2-0.3.1-alpha-2_.jar             |TerraForged                   |terraforged                   |0.3.1               |DONE      |Manifest: NOSIGNATURE
		RubidiumExtra-1.18.2-0.4.9.13.jar                 |Rubidium Extra                |sodiumextra                   |1.18.2-0.4.9.13     |DONE      |Manifest: NOSIGNATURE
		Placebo-1.18.2-6.5.1.jar                          |Placebo                       |placebo                       |6.5.1               |DONE      |Manifest: NOSIGNATURE
		StorageDrawers-1.18.2-10.2.1_.jar                 |Storage Drawers               |storagedrawers                |10.2.1              |DONE      |Manifest: NOSIGNATURE
		lazydfu-1.0-1.18+_.jar                            |LazyDFU                       |lazydfu                       |0.1.3               |DONE      |Manifest: NOSIGNATURE
		ferritecore-4.2.1-forge_.jar                      |Ferrite Core                  |ferritecore                   |0.0NONE             |DONE      |Manifest: 41:ce:50:66:d1:a0:05:ce:a1:0e:02:85:9b:46:64:e0:bf:2e:cf:60:30:9a:fe:0c:27:e0:63:66:9a:84:ce:8a
		engineersdecor-1.18.2-1.1.24_.jar                 |Engineer's Decor              |engineersdecor                |1.1.24              |DONE      |Manifest: bf:30:76:97:e4:58:41:61:2a:f4:30:d3:8f:4c:e3:71:1d:14:c4:a1:4e:85:36:e3:1d:aa:2f:cb:22:b0:04:9b
		JadeAddons-1.18.2-2.4.0_.jar                      |Jade Addons                   |jadeaddons                    |2.4.0               |DONE      |Manifest: NOSIGNATURE
		framework-0.2.4-1.18.2_.jar                       |Framework                     |framework                     |0.2.4               |DONE      |Manifest: NOSIGNATURE
		rubidium_extras-1.18.2_v1.3.2.jar                 |Rubidium Extras               |rubidium_extras               |1.18.2_v1.3.2       |DONE      |Manifest: NOSIGNATURE
		forge-1.18.2-40.1.73-universal.jar                |Forge                         |forge                         |40.1.73             |DONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		rankine-1.18.2-1.3.5_.jar                         |Project Rankine               |rankine                       |1.3.5               |DONE      |Manifest: NOSIGNATURE
		ImmersivePosts-1.18.2-4.3.0-3_.jar                |Immersive Posts               |immersiveposts                |4.3.0-3             |DONE      |Manifest: NOSIGNATURE
		ImmersiveEngineering-1.18.2-8.1.0-150_.jar        |Immersive Engineering         |immersiveengineering          |1.18.2-8.1.0-150    |DONE      |Manifest: NOSIGNATURE
		ironchest-1.18.2-13.1.9_.jar                      |Iron Chests                   |ironchest                     |1.18.2-13.1.9       |DONE      |Manifest: NOSIGNATURE
		corpse-1.18.2-1.0.1_.jar                          |Corpse                        |corpse                        |1.18.2-1.0.1        |DONE      |Manifest: NOSIGNATURE
		createaddition-1.18.2-20220715a_.jar              |Create Crafts & Additions     |createaddition                |1.18.2-20220715a    |DONE      |Manifest: NOSIGNATURE
	Flywheel Backend: GL33 Instanced Arrays
	Crash Report UUID: 4d81a79f-b9e5-4700-82a7-89c6d44dda0c
	FML: 40.1
	Forge: net.minecraftforge:40.1.73
```

**Comment by CannoliCatfish** [comment]
This is a known issue and will be fixed in the next update coming soon. A temporary solution is using a different block (that is not a sheetmetal block), placing the water, and then replacing that block with the sheetmetal block.

**Comment by CannoliCatfish** [comment]
Fixed as of the most recent release for 1.18.2 (Version 1.3.6).
